src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -48,8 +48,13 @@ public static void MapBackground(IImageHandler handler, IImage image)
 		public static void MapSource(IImageHandler handler, IImage image) =>
 			MapSourceAsync(handler, image).FireAndForget(handler);
 
-		public static Task MapSourceAsync(IImageHandler handler, IImage image) =>
-			handler.SourceLoader.UpdateImageSourceAsync();
+		public static Task MapSourceAsync(IImageHandler handler, IImage image)
```

Comment:
Should we just make this an `async Task` and then await each? Not sure if this may have issues with the ContinueWith... Are we sure the threads will be correct when the source finishes?